dash/test/dart/DARTMemAllocTest.cc

```diff
@@ -169,3 +169,42 @@ TEST_F(DARTMemAllocTest, SegmentReuseTest)
     DART_OK,
     dart_team_memfree(gptr2));
 }
+
+
+TEST_F(DARTMemAllocTest, AllocatorTest)
```

Comment:
Can we test allocation / deallocation for various pool-sizes, including impossible allocations. AFAIK the buddy allocation is not extensively tested yet.

We should also consider fragmentation.